### ISSUE TITLE
tf2onnx.convert failure

### ISSUE THREAD

**Issue by kaustav-aarish** [onnx/onnx#2885]
# Bug Report

### Describe the bug
Hello, I made a script which converts a TensorFlow saved file into onnx. This was working perfectly, until I noticed a sudden error when running it today: 

    [libprotobuf ERROR google/protobuf/descriptor_database.cc:394] Invalid file descriptor data passed to 
    EncodedDescriptorDatabase::Add().
    [libprotobuf FATAL google/protobuf/descriptor.cc:1356] CHECK failed: GeneratedDatabase()- 
    >Add(encoded_file_descriptor, size): 
    libc++abi.dylib: terminating with uncaught exception of type google::protobuf::FatalException: CHECK failed: 
    GeneratedDatabase()->Add(encoded_file_descriptor, size): 
    Abort trap: 6

The command that was called was this:

    python -m tf2onnx.convert --saved-model model --output ./model.onnx --opset 7

Following this error, this always pops us:

![image](https://user-images.githubusercontent.com/65616588/86816616-f1e44200-c07b-11ea-860e-62f23d286f11.png)

### Further information
I have followed tried to follow the suggestions of the following related issues:

https://github.com/tensorflow/tensorflow/issues/33746
https://github.com/tensorflow/tensorflow/issues/35573

Specifically, switching the protobuf version to 3.8 seems to be a popular solution, however, none have worked for me.

I don't believe it is a model-specific problem, but it was a network made for mnist and used opset=7 as shown in the called bash command.

### System information
- OS Platform and Distribution: macOS Catalina Version 10.15.5
- ONNX version:  1.7.0
- Python version: 3.6.0
- GCC/Compiler version (if compiling from source):
- CMake version:
- Protobuf version: 3.8.0

### Reproduction
This was the script to produce the model:

    import tensorflow.compat.v2 as tf
    import tensorflow_datasets as tfds

    tfds.disable_progress_bar()
    tf.enable_v2_behavior()

    (ds_train, ds_test), ds_info = tfds.load(
        'mnist',
        split=['train', 'test'],
        shuffle_files=True,
        as_supervised=True,
        with_info=True,
    )

    def normalize_img(image, label):
    """Normalizes images: `uint8` -> `float32`."""
    return tf.cast(image, tf.float32) / 255., label

    ds_train = ds_train.map(
        normalize_img, num_parallel_calls=tf.data.experimental.AUTOTUNE)
    ds_train = ds_train.cache()
    ds_train = ds_train.shuffle(ds_info.splits['train'].num_examples)
    ds_train = ds_train.batch(128)
    ds_train = ds_train.prefetch(tf.data.experimental.AUTOTUNE)

    ds_test = ds_test.map(
        normalize_img, num_parallel_calls=tf.data.experimental.AUTOTUNE)
    ds_test = ds_test.batch(128)
    ds_test = ds_test.cache()
    ds_test = ds_test.prefetch(tf.data.experimental.AUTOTUNE)

    model = tf.keras.models.Sequential([
    tf.keras.layers.Flatten(input_shape=(28, 28, 1)),
    tf.keras.layers.Dense(128,activation='relu'),
    tf.keras.layers.Dense(10, activation='softmax')
    ])
    model.compile(
        loss='sparse_categorical_crossentropy',
        optimizer=tf.keras.optimizers.Adam(0.001),
        metrics=['accuracy'],
    )

    model.fit(
        ds_train,
        epochs=6,
        validation_data=ds_test,
    )
    model.save('model')
And then, this command was run in the same directory as the script above:

    python -m tf2onnx.convert --saved-model model --output ./model.onnx --opset 7

### Expected behavior

Normally, this is the output I get (this is a screenshot from a Linux environment with different specs but provides the same output):

    WARNING:tensorflow:From /home/zilun/anaconda3/envs/tensorflow2/lib/python3.8/site- 
    packages/tf2onnx/tf_loader.py:324: extract_sub_graph (from tensorflow.python.framework.graph_util_impl) is deprecated 
    and will be removed in a future version.
    Instructions for updating:
    Use `tf.compat.v1.graph_util.extract_sub_graph`
    2020-07-07 17:36:42,007 - WARNING - From /home/zilun/anaconda3/envs/tensorflow2/lib/python3.8/site- 
    packages/tf2onnx/tf_loader.py:324: extract_sub_graph (from tensorflow.python.framework.graph_util_impl) is deprecated 
    and will be removed in a future version.
    Instructions for updating:
    Use `tf.compat.v1.graph_util.extract_sub_graph`
    2020-07-07 17:36:42,031 - INFO - Using tensorflow=2.2.0, onnx=1.7.0, tf2onnx=1.6.2/8d5253
    2020-07-07 17:36:42,031 - INFO - Using opset <onnx, 7>
    2020-07-07 17:36:42,079 - INFO - Optimizing ONNX model
    2020-07-07 17:36:42,093 - INFO - After optimization: Cast -1 (1->0), Identity -5 (5->0)
    2020-07-07 17:36:42,094 - INFO - 
    2020-07-07 17:36:42,094 - INFO - Successfully converted TensorFlow model model to ONNX
    2020-07-07 17:36:42,096 - INFO - ONNX model is saved at ./model.onnx

**Comment by askhade** [comment]
@kaustav-aarish : Please open this bug in tf2onnx converter repo
